.changes/unreleased/tinymce-TINY-10602-2024-04-04.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Theme loader now respects the suffix when loading skin CSS files
```

Looks like a bug fix so maybe past tense is better:
```suggestion
body: Theme loader did not respect the suffix when it was loading skin CSS files.
```

Also I think we should have a `.` a the end.